src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const everyDayCarRentCost = 40;
+  let discount = 0;
+
+  if (days >= 3 && days < 7) {
+    discount = 20;
```

Comment:
Use constants for the minimum number of days and the discount amount to avoid hardcoding these values. This will make the code more readable and maintainable.